inventory/webhook.py

```
@@ -0,0 +1,36 @@
+import requests
+import logging
+from decouple import config
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+TOKEN = config("TELEGRAM_TOKEN") #'6879105065:AAE68ylp86mMmPnI8z9b41NKaR83vo4GgOE' 
+CHAT_ID = '-1002084597891'
+
+def telebot_send_text(text, photo, topic):
+    TOPICS = {
+        "Deletion": "2",  
+        "Expiry/Low qty": "13",  
+        "Orders": "15",  
+        "Loans & Returns": "8",  
+    }
+    
+    if topic in TOPICS:
```

maybe we should raise an exception if it is not in `TOPICS`